intdataplane/int_dataplane.go

```diff
@@ -326,6 +334,15 @@ func NewIntDataplaneDriver(config Config) *InternalDataplane {
 		dp.allIptablesTables = append(dp.allIptablesTables, t)
 	}
 
+	// Register that we will report a liveness indicator.
+	if config.HealthState != nil {
+		config.HealthState.RegisterReporter(
+			HEALTH_NAME,
+			&health.HealthReport{Live: true, Ready: true},
+			HEALTH_INTERVAL*2,
```

Comment:
As discussed, we already have 60s here, with the k8s default failureThreshold - so leaving this as is.